dts/bindings/sensor/ti,tmag5273.yaml

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2023, deveritec GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Texas Instruments TMAG5273 Low-Power Linear 3D Hall-Effect Sensor with an I2C interface.
+  See the specification for the default I2C address.
+  The specification of the sensor can be found at:
+    https://www.ti.com/lit/ds/symlink/tmag5273.pdf
+  The <zephyr/dt-bindings/sensor/tmag5273.h> file should be included in the
+  DeviceTree and it provides macros that can be used for configuration.
+
+compatible: "ti,tmag5273"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  operation-mode:
+    type: int
+    default: 0
+    enum:
+      - 0
+      - 1
+    description: |
+      Sensor mode used if set to "active".
+      Defaults to continuous sampling.
+
+        0 # continuous
+        1 # standby
+
+  axis:
+    type: int
+    default: 0x7
```

Comment:
Isn't the justification on line 47 good enough?
The reset default is "measurement on no axis", though from my perspective it would make more sense to configure the sensor in a more "running by default" way without adding properties in the board dts.